make_prg/io_utils.py

```diff
@@ -5,11 +5,12 @@
 
 from Bio import AlignIO
 
-from make_prg import MSA
+from make_prg.from_msa import MSA
 from make_prg.prg_encoder import PrgEncoder, PRG_Ints
 
 
 def load_alignment_file(msa_file: str, alignment_format: str) -> MSA:
+    msa_file = str(msa_file)
     logging.info("Read from MSA file %s", msa_file)
     if ".gz" in msa_file:
```

Comment:
```suggestion
    if msa_file.endswith(".gz"):
```
Is probably much safer

Reply:
adding this!